src/definitions/globals/site.js

```diff
@@ -10,6 +10,13 @@
 
 ;(function ($, window, document, undefined) {
 
+$.isFunction = $.isFunction || function(obj) {
```

Comment:
Does this work if we use single component ?

Reply:
Well, if you just include single component files, then currently you would need to have site.js aswell, yes.
I was trying not to invent another `wrapper_functions.js` or similiar, but use existing code.
As it is needed to ensure those functions are available before any other FUI code is loaded, site.js was my only option because it is compiled at the very beginning already.
Otherwise we would need to put those function into every single component, which is wrong and overkill in my pov. 
As long as we either rely on jquery or some wrapper of jquery functions into our own helper file (to avoid having big changes in the current code) then you always need to include at least site.js (or a new `helper/wrapper.js`) just as you do with jquery.

Reply:
Unfortunately for now we'll have to put those functions in each module, to keep 100% compatibility with SUI, even if, and i agree with you, it's overkill (we already do it with debug functions).

Reply:
I moved/copied the function declaration accordingly now. See latest commit